server/src/domain/media/media.service.ts

```diff
@@ -334,4 +335,13 @@ export class MediaService {
     this.storageRepository.mkdirSync(folderPath);
     return join(folderPath, `${asset.id}.${extension}`);
   }
+
+  isSRGB(asset: AssetEntity): boolean {
+    const { colorspace, profileDescription, bitsPerSample } = asset.exifInfo ?? {};
+    if (colorspace || profileDescription) {
+      return [colorspace, profileDescription].includes('sRGB');
```

Comment:
Is this case sensitive and does it need to be?

Reply:
The value for `ColorSpace` is numeric and parsed into `sRGB` based on the below EXIF spec, so it should always have that casing.

```
0x1 = sRGB
0x2 = Adobe RGB
0xfffd = Wide Gamut RGB
0xfffe = ICC Profile
0xffff = Uncalibrated
```

But the profile description can be anything, so this should actually just search for `srgb` in the strings uncased.

Reply:
Great, looks good.